src/index.js

```diff
@@ -32,6 +32,8 @@ const theme = {
   foreground: 'white',
   gray: '#3f4e60',
   grayAlt: '#222e3e',
+  inputBackgroundColor: '#fff',
+  inputTextColor: '#000',
```

Comment:
If we ever have a bright theme, we probably won’t want to make inputs with a black background. Therefore, it is necessary to create two new theme vars instead of reusing `foreground` / `background`, for instance.